gym/envs/box2d/lunar_lander.py

```diff
@@ -121,15 +121,18 @@ class LunarLander(gym.Env, EzPickle):
 
     ## Arguments
     To use to the _continuous_ environment, you need to specify the
-    `continuous"=True` argument like below:
+    `continuous"=True"` argument like below:
```

Comment:
Looks like something is wrong here, I guess it should be:
```markdown
`continuous=True` argument like below:
```

Reply:
Thanks for the catch